src/api/Controllers/UploadController.cs

```diff
@@ -147,6 +153,54 @@ public async Task<ActionResult<int>> UploadFileAsync(int workgroupId, IFormFile
                 }
             }
 
+            // Add lithology imports if provided
+            if (lithologyImports != null)
```

Comment:
PP: Wir sollten die Import-Logik für Boreholes und Lithos vermutlich in separate Methoden auslagern, analog zu den Validierungs-Methoden. Langsam aber sicher wird die Controller-Methode zu lang und unübersichtlich. Die Extrahierung in separate Methoden wäre ein erster Schritt, in einem nächsten könnten wir die Import- und Validierungslogik in separate Klassen extrahieren. Würde das Testing vermutlich auch vereinfachen.

Können dieses Thema aber gerne noch gemeinsam anschauen. Das Refactoring kann in einem separaten PR geschehen.

Reply:
Ja können das gerne nächste Woche noch anschauen. Ich denke das könnte Teil von diesem task sein oder?
#525